runtime/src/main/java/org/corfudb/runtime/object/transactions/WriteSetSMRStream.java

```diff
@@ -160,14 +161,14 @@ void mergeTransaction() {
     public List<SMREntry> previous() {
         writePos--;
 
-        if (writePos <= Address.NEVER_READ) {
-            writePos = Address.NEVER_READ;
+        if (writePos <= Address.maxNonAddress()) {
```

Comment:
should this check be a utility of the Address class ? Why do  we need to expose this ?